test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -79,6 +79,41 @@ def func(tensor):
         tensor = common_utils.get_whitenoise()
         self._assert_consistency(func, tensor)
 
+    def test_inverse_spectrogram_complex(self):
+        def func(tensor):
+            length = 400
+            n_fft = 400
+            ws = 400
+            hop = 200
+            pad = 0
+            window = torch.hann_window(ws, device=tensor.device, dtype=torch.view_as_real(tensor).dtype)
+            power = None
+            normalize = False
+            return F.inverse_spectrogram(tensor, length, pad, window, n_fft, hop, ws, power, normalize)
+
+        waveform = common_utils.get_whitenoise(sample_rate=8000, duration=0.05)
+        tensor = F.spectrogram(waveform, window=torch.hann_window(400, device=waveform.device, dtype=waveform.dtype),
```

Comment:
It's ideal if each unit test uses only the function to be tested. Say if a bug is introduced in `F.spectrogram`, then this test might also fail, which is a false signal.

There is [`get_spectrogram`](https://github.com/pytorch/audio/blob/16f3b2f943237f883956e82e3d71cf99622226b9/test/torchaudio_unittest/common_utils/data_utils.py#L115) function, which practically does the same thing as `F.spectrogarm`.